api_contracts/contracts/search/AdvancedSearch.md

```diff
@@ -29,9 +29,9 @@ This endpoint returns a list of the top results of specific categories sorted by
             + `naics`
             + `county`
             + `district`
-            + `federal_accounts`
-            + `countries`
-            <!-- + `states_and_territories` -->
+            + `federal_account`
+            + `country`
+            <!-- + `state_territory` -->
```

Comment:
Can this be uncommented?

Reply:
Ah, I commented this out to test the error messages